apps/stripe/src/modules/app-config/app-config.ts

```diff
@@ -0,0 +1,25 @@
+import { env } from "@/lib/env";
+
+interface AppConfigStorage {
+  getSecretKeyValue(): string;
+  getPublishableKeyValue(): string;
+}
+
+export class EnvAppConfigStorage implements AppConfigStorage {
+  getSecretKeyValue(): string {
+    return env.STRIPE_SECRET_KEY;
+  }
+  getPublishableKeyValue(): string {
+    return env.STRIPE_PUBLISHABLE_KEY;
+  }
+}
+
+export class AppConfig {
+  constructor(private deps: { storage: AppConfigStorage }) {}
+  getStripeSecretKeyValue(): string {
+    return this.deps.storage.getSecretKeyValue();
+  }
+  getStripePublishableKey(): string {
+    return this.deps.storage.getPublishableKeyValue();
+  }
```

Comment:
IMO we should return here value object, not value.

In general data should flow throw app as value objects, which is safe, only when it's needed, we decode it (eg when making a http call)